generators/PHPythia8/PHPythia8.C

```diff
@@ -71,9 +74,9 @@ int PHPythia8::Init(PHCompositeNode *topNode)
 {
   if (!_configFile.empty()) read_config();
   for (unsigned int j = 0; j < _commands.size(); j++)
-  {
-    _pythia->readString(_commands[j]);
-  }
+    {
+      _pythia->readString(_commands[j]);
```

Comment:
This line should fail since `_pythia = 0` here.  Probably we have to use both `ppGen` and `pnGen`.